src/main/java/vendored/org/apache/dolphinscheduler/common/utils/process/ProcessImplForWin32.java

```diff
@@ -210,8 +198,7 @@ static Process start(String username,
                 } else if (redirects[1] == ProcessBuilderForWin32.Redirect.INHERIT) {
                     stdHandles[1] = getHandle(FileDescriptor.out);
                 } else {
-                    f1 = newFileOutputStream(redirects[1].file(),
-                            redirects[1].append());
+                    f1 = newFileOutputStream(redirects[1].file(), redirects[1].append());
                     stdHandles[1] = getHandle(f1.getFD());
                 }
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Similar code fragments were detected in the same file at the following lines: 196:203, 205:212.  
Refactoring can help improve code maintainability. Consider reducing duplicate code by extracting it into a separate method. You can then replace duplicated code with calls to this new method.